packages/conf-pkg-config/conf-pkg-config.1.2/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "unixjunkie@sdf.org"
+authors: ["Francois Berenger"]
+homepage: "http://www.freedesktop.org/wiki/Software/pkg-config/"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "GPL"
+build: [
+  ["pkg-config" "--help"]
+]
+depends: [
+  "pkg-config-openbsd" {os = "openbsd"}
+]
+depexts: [
+  ["pkg-config"] {os-distribution = "debian"}
+  ["pkg-config"] {os-distribution = "ubuntu"}
+  ["pkg-config"] {os-distribution = "archlinux"}
+  ["pkgconfig"] {os-distribution = "fedora"}
+  ["pkgconfig"] {os-distribution = "centos"}
+  ["pkgconfig"] {os-distribution = "mageia"}
+  ["pkgconfig"] {os-distribution = "rhel"}
+  ["pkgconfig"] {os-distribution = "oraclelinux"}
+  ["pkgconfig"] {os-distribution = "alpine"}
+  ["devel/pkgconf"] {os = "freebsd"}
+  ["pkg-config"] {os = "macos" & os-distribution = "homebrew"}
+  ["pkgconf"] {os = "freebsd"}
+]
+synopsis: "Virtual package relying on pkg-config installation"
+description: """
+This package can only install if the pkg-config package is installed
+on the system."""
+flags: light-uninstall
```

Comment:
I think the wanted flag here would be `conf`? cc @rjbou

Reply:
I changed it.

Reply:
@kit-ty-kate yes! it's juste the wrong linting that complains about `light_uninstall` flag